src/main/java/fr/arthurbambou/fdlink/discordstuff/DiscordBot.java

```diff
@@ -166,40 +204,51 @@ public DiscordBot(String token, FDLink.Config config) {
 
                 this.hasReceivedMessage = false;
             }
-            if (this.hasChatChannels && (this.config.minecraftToDiscord.chatChannels.customChannelDescription ||  this.config.minecraftToDiscord.logChannels.customChannelDescription) && this.ticks >= 200) {
-                this.ticks = 0;
+/*             if ((this.hasChatChannels || this.hasLogChannels) && (this.config.minecraftToDiscord.chatChannels.customChannelDescription ||  this.config.minecraftToDiscord.logChannels.customChannelDescription) && ((Util.getMeasuringTimeMs()/1000) % 10) == 0) {
                 int totalUptimeSeconds = (int) (Util.getMeasuringTimeMs() - this.startTime) / 1000;
                 final int uptimeH = totalUptimeSeconds / 3600 ;
                 final int uptimeM = (totalUptimeSeconds % 3600) / 60;
                 final int uptimeS = totalUptimeSeconds % 60;
 
-                for (String id : this.config.chatChannels) {
-                    this.api.getServerTextChannelById(id).ifPresent(channel ->
-                            channel.updateTopic(String.format(
-                            "player count : %d/%d,\n" +
-                                    "uptime : %d h %d min %d second",
-                            playerNumber, maxPlayer, uptimeH, uptimeM, uptimeS
+                if(this.config.minecraftToDiscord.chatChannels.customChannelDescription){
```

Comment:
For now it's good to have that removed since you could be API banned by Discord.
However we should look at reimplementing it with a proper delay (#92 discussion)

Reply:
Doing 5 or 10 Minutes is an easy fix. I will readd the feature since it seems people still want it, even though it's not accurate information.